collector/host_inventory.py

```diff
@@ -0,0 +1,109 @@
+import base64
+import json
+import logging
+import math
+
+from threading import current_thread
+
+import prometheus_metrics
+from . import utils
+from .env import HOST_INVENTORY_HOST, HOST_INVENTORY_PATH
+
+LOGGER = logging.getLogger()
+URL = f'{HOST_INVENTORY_HOST}/{HOST_INVENTORY_PATH}'
+
+
+def _retrieve_hosts(headers: dict) -> dict:
+    """Collect all hosts for account.
+
+    Parameters
+    ----------
+    headers (dict)
+        HTTP Headers that will be used to request data
+
+    Returns
+    -------
+    dict
+        Host collection
+
+    """
+    url = URL + '/hosts?per_page=50&page={}'
```

Comment:
since you've removed the `HOST_INVENTORY_PER_PAGE` env, you can remove this `per_page=50` as well.

Reply:
Ok, are other values supported? Can we lower the request count by increasing this value? What's the I think 50 entries is the default now, right? If so, maybe increasing this to 100 would make less overhead for us?

Reply:
yes, in my test, 50 is the default and the highest it accepts is 100.
But this can be changed by the API side, so not a good idea to hard code it here. We can either 
1) always use the default (i.e. not specifying)
2) allow an env to adjust this

Reply:
Let's not stretch it more than needed and use the default then.

Reply:
sounds good to me